### PR TITLE
Refactor unverified contracts to use `promiseAllPlus`

### PR DESCRIPTION
**Pull request by krzkaczor**

### CODE REVIEW

File: packages/config/scripts/checkVerifiedContracts/tasks.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
-import { EtherscanClient, Logger, TaskQueue } from '@l2beat/common'
+import {
+  EtherscanClient,
+  Logger,
+  promiseAllPlus,
+  TaskQueue,
```
Comment (adamiak) [inline packages/config/scripts/checkVerifiedContracts/tasks.ts:5]: `TaskQueue` is no longer used.

File: packages/config/scripts/checkVerifiedContracts/tasks.ts [hunk 2]
```diff
@@ -13,67 +18,22 @@ export async function verifyContracts(
...
-      workers: workersCount,
-      shouldRetry: (_) => {
-        queueStatus.errorOccurred = true
-        return { retry: false }
-      },
-    },
-  )
-
-  return { taskQueue, queueStatus }
+  return Object.fromEntries(verification)
```
Comment (adamiak) [inline packages/config/scripts/checkVerifiedContracts/tasks.ts:38]: My IDE screams that `verification` has `any` type. I'm not sure why our lint doesn't pick it up here.

**Review by adamiak — APPROVED** [review]

File: packages/config/scripts/checkVerifiedContracts/tasks.ts (hunk 2 above)
Comment (krzkaczor) [reply packages/config/scripts/checkVerifiedContracts/tasks.ts:38]: rebuild the project, perhaps. Works for me (TM)

Comment (adamiak) [reply packages/config/scripts/checkVerifiedContracts/tasks.ts:38]: Restarting IDE helped :/

**Pull request merged**